docs/changehistory/NextVersion.md

```diff
@@ -94,6 +96,32 @@ The Presentation system is moving towards a more modular approach, with smaller
 
 The unified selection system has been part of `@itwin/presentation-frontend` for a long time, providing a way for apps to have a single source of truth of what's selected. This system is now deprecated in favor of the new [@itwin/unified-selection](https://www.npmjs.com/package/@itwin/unified-selection) package. See the [migration guide](https://github.com/iTwin/presentation/blob/master/packages/unified-selection/learning/MigrationGuide.md) for migration details.
 
+## Map layers
+
+### Google Maps 2D tiles API
+
+The `itwin\map-layers-formats` package now includes an API for consuming Google Maps 2D tiles.
+
+To enable it as a base map, it's simple as:
+
+ ```typescript
+import { GoogleMaps } from "@itwin/map-layers-formats";
+const ds = IModelApp.viewManager.selectedView.displayStyle;
```

Comment:
Please consider putting sample code into the example-code-snippets package, to ensure it compiles now and in the future and can be referenced from the API docs as well. (Your example here has a compilation error, for example). This also allows you to provide a more complete example for someone to quickly integrate Google Maps into their own app.

Reply:
Will do.
I have seen example-code being referenced in .md files, can you do the same with JSDoc's @example ?

Reply:
[Here's an example](https://github.com/iTwin/itwinjs-core/blob/7e3c03b4c5027e5a4085a5ed42b3d9f34d338cee/core/frontend/src/tile/RealityTileTree.ts#L140) of referencing example code from API docs.

Reply:
done